setup.py

```diff
@@ -29,8 +29,8 @@
 
 head_dims = os.environ.get("FLASHINFER_HEAD_DIMS", "64,128,256").split(",")
 head_dims = list(map(int, head_dims))
-SM90_ALLOWED_HEAD_DIMS = {64, 128, 256}
-head_dims_sm90 = [d for d in head_dims if d in SM90_ALLOWED_HEAD_DIMS]
+SM90_ALLOWED_HEAD_DIMS = {(64, 64), (128, 128), (256, 256), (192, 128)}
```

Comment:
(192, 128) is not compatible with page attention kernels, can we separate head_dim for page attention and ragged attention kernels?